### PR TITLE
Build e2e-tests image in appstudio.

### PR DESCRIPTION
**Pull request by rhopp**
Signed-off-by: Radim Hopp <rhopp@redhat.com>

### CODE REVIEW

File: components/e2e-tests/.tekton/serviceaccount.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: pipeline
+secrets:
+  - name: redhat-appstudio-staginguser-pull-secret
```
Comment (Michkov) [inline components/e2e-tests/.tekton/serviceaccount.yaml:6]: you should also create e2e-tests-ci.yaml under components/authentication to give permission to create this secret.

Comment (rhopp) [reply components/e2e-tests/.tekton/serviceaccount.yaml:6]: Thanks! That's the piece of the puzzle I was missing... When I tried it on my cluster I created this secret manually with my personal creds, but I had no idea how is this handled on our staging cluster...

Comment (rhopp) [reply components/e2e-tests/.tekton/serviceaccount.yaml:6]: @Michkov I've added QE team as pipeline-maintainers for e2e-tests namespace. PTAL

Comment (Michkov) [reply components/e2e-tests/.tekton/serviceaccount.yaml:6]: Include the file in components/authentication/kustomization.yaml

**Review by Michkov — APPROVED** [review]

File: components/e2e-tests/.tekton/serviceaccount.yaml (hunk 1 above)
Comment (rhopp) [reply components/e2e-tests/.tekton/serviceaccount.yaml:6]: I feel so dumb right now... Fixed.

**Review by Michkov — APPROVED** [review]

**Pull request merged**